modules/statics/src/coins.ts

```diff
@@ -1252,4 +1254,136 @@ export const coins = CoinMap.fromCoins([
     '0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e',
     UnderlyingAsset['avaxc:USDC']
   ),
+  solToken(
+    'sol:SRM',
+    'Serum',
+    6,
+    'SRMuApVNdxXokk5GT7XD5cUUgXMBCoAz2LHeuAoKWRt', // https://explorer.solana.com/address/SRMuApVNdxXokk5GT7XD5cUUgXMBCoAz2LHeuAoKWRt
+    'SRMuApVNdxXokk5GT7XD5cUUgXMBCoAz2LHeuAoKWRt',
+    UnderlyingAsset.SRM,
+    AccountCoin.DEFAULT_FEATURES,
+    '',
+    'SRM'
+  ),
+  solToken(
+    'sol:ORCA',
+    'Orca',
+    6,
+    'orcaEKTdK7LKz57vaAYr9QeNsVEPfiu6QeMU1kektZE', // https://explorer.solana.com/address/orcaEKTdK7LKz57vaAYr9QeNsVEPfiu6QeMU1kektZE
+    '8DzsCSvbvBDYxGB4ytNF698zi6Dyo9dUBVRNjZQFHSUt',
+    UnderlyingAsset.ORCA,
+    AccountCoin.DEFAULT_FEATURES,
+    '',
+    'ORCA'
+  ),
+  solToken(
+    'sol:USDC',
+    'USD Coin',
+    6,
+    'EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v', // https://explorer.solana.com/address/EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v
+    '2wmVCSfPxGPjrnMMn7rchp4uaeoTqN39mXFC2zhPdri9',
+    UnderlyingAsset.USDC,
+    AccountCoin.DEFAULT_FEATURES,
+    '',
+    'USDC'
+  ),
+  solToken(
+    'sol:RAY',
+    'Raydium',
+    6,
+    '4k3Dyjzvzp8eMZWUXbBCjEvwSkkk59S5iCNLY3QrkX6R', // https://explorer.solana.com/address/4k3Dyjzvzp8eMZWUXbBCjEvwSkkk59S5iCNLY3QrkX6R
+    '7nJGERf1LZpjn4uTfcDZLJ6YRtvdSExAB9uK851zT6do',
+    UnderlyingAsset.RAY,
+    AccountCoin.DEFAULT_FEATURES,
+    '',
+    'RAY'
+  ),
+  solToken(
+    'sol:MNDE',
+    'Marinade',
+    9,
+    'MNDEFzGvMt87ueuHvVU9VcTqsAP5b3fTGPsHuuPA5ey', // https://explorer.solana.com/address/MNDEFzGvMt87ueuHvVU9VcTqsAP5b3fTGPsHuuPA5ey/metadata
+    'MNDEFzGvMt87ueuHvVU9VcTqsAP5b3fTGPsHuuPA5ey',
+    UnderlyingAsset.MNDE,
+    AccountCoin.DEFAULT_FEATURES,
+    '',
+    'MNDE'
+  ),
+  solToken(
+    'sol:SLND',
+    'Solend',
+    6,
+    'SLNDpmoWTVADgEdndyvWzroNL7zSi1dF9PC3xHGtPwp', // https://explorer.solana.com/address/SLNDpmoWTVADgEdndyvWzroNL7zSi1dF9PC3xHGtPwp
+    'EsLAEeKA1dUJRiPpbTAm7r94KqGDAKe1ivH8PLAAuSxV',
+    UnderlyingAsset.SLND,
+    AccountCoin.DEFAULT_FEATURES,
+    '',
+    'SLND'
+  ),
+  tsolToken(
+    'tsol:SRM',
+    'Serum',
+    6,
+    'Test SRM',
+    'Test Mint',
```

Comment:
we should create our own mints for these tokens in devnet (in sandbox) and refer to them here.

this way, we will be able to do internal testing in dev, test, testnet environments